styles/posts.css

```diff
@@ -0,0 +1,58 @@
+main {
+    padding-top: 0;
+}
+
+#error-display {
+    color: red;
+}
+
+.post {
+    display: grid;
+    grid-template-columns: 120px 1fr;
+    grid-template-rows: 30px 60px;
+
+    column-gap: 20px;
+    background: var(--offset-color);
+    box-shadow: var(--box-shadow);
+}
+
+
+#post-list {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(275px, 1fr));
+    grid-auto-rows: 400px;
+    gap: 20px;
+}
+
+li {
+    background: var(--offset-color);
+    box-shadow: var(--box-shadow);
+    transition: box-shadow 250ms ease-out;
+}
+
+li:hover {
+    box-shadow: var(--hover-box-shadow);
```

Comment:
...which is why nothing happens on hover. but good job thinking of UI that responds to user actions